src/utils/streams.js

```diff
@@ -0,0 +1,121 @@
+import program from 'commander'
+import { validateOnDefinitions, validateFileExtension, validateOnFileExist } from './validator'
+import readline from 'readline'
+import fs from 'fs'
+import csv from 'csvtojson'
+
+var reader = readline.createInterface({
+    input: process.stdin,
+    output: process.stdout,
+    terminal: false
+});
+
+const commands = {
+    reverse: function reverse(str) {
```

Comment:
Actually, there's no need for writing `methodName: function() {}`, it's a bit outdated although it's relevant. You can use one of these  syntaxes:
1. `methodName: (params) => {}` in case `this` is insufficient for you
2. `methodName(params) {}` - most common syntax for method declarations